src/toxcore/dht/server.rs

```diff
@@ -982,4 +1004,22 @@ mod tests {
         let (_, lan_discovery) = LanDiscovery::from_bytes(&buf[..size]).unwrap();
         assert_eq!(alice.pk, lan_discovery.pk);
     }
+    // remove_timedout_client()
+    #[test]
+    fn server_remove_timedout_client_test() {
+        let (alice, _precomp, _bob_pk, _bob_sk, _rx, _addr) = create_node();
+        let node = PackedNode::new(false, SocketAddr::V4("127.0.0.1:12345".parse().unwrap()), &alice.pk);
+
+        alice.try_add_to_kbucket(&node);
+
+        alice.send_pings().wait().unwrap();
+        // timeout is 122 seconds
+        let dur = time::Duration::from_secs(123);
+
+        thread::sleep(dur);
```

Comment:
I don't think we want to have tests that will run more than 2 minutes. It's better to change client timestamp in the test.